<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>691</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>State = Configuring</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="configure">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="storeConfig">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>storeConfig</string>
    </property>
   </widget>
   <widget class="QPushButton" name="programStart">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>programStart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="armThesystem">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>armTheSystem</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doorOpened">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>doorOpened</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UserGaveCorrectPin">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>UserGaveCorrectPIn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UserGaveWrongPin">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>UserGaveWrongPin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testiButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>420</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>TestiButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
